packages/css-library/src/utilities/display.scss

```diff
@@ -0,0 +1,14 @@
+@use 'uswds-core/src/styles/functions' as *;
+@use '../tokens/display' as *;
+@use './settings' as *;
+
+$u-display: (
+  display: (
+    base: 'vads',
```

Comment:
Just having the `vads` base with tokens like `none` and `flex` means we will have utility names like this for setting `display`:

- `vads-none`
- `vads-flex`

We'll also have utilities like `vads-flex-1` for setting the `flex` property, but it seems fine to have the similar names since they're related, and I don't think things will be overridden which is good.